src/common/pf_lldp.c

```diff
@@ -493,6 +491,19 @@ static void pf_lldp_reset_peer_timeout (pnet_t * net, uint16_t timeout_in_secs)
    }
 }
 
+void pf_lldp_get_port_config (
+   pnet_t * net,
+   int loc_port_num,
+   const pnet_lldp_port_cfg_t ** pp_port_cfg)
+{
+   if (loc_port_num < 1 || loc_port_num >= PNET_MAX_PORT)
```

Comment:
```suggestion
   if (loc_port_num < 1 || loc_port_num > PNET_MAX_PORT)
```

Reply:
Thanks.
This also shadowed a missing return statement.
Done.